rd_ui/app/scripts/services/notifications.js

```diff
@@ -27,23 +30,13 @@
                 return;
             }
 
-            if (document.webkitVisibilityState && document.webkitVisibilityState == 'visible') {
```

Comment:
This is here to make sure we don't show notifications when the page is in focus. Is there a reason you removed it?